### ISSUE TITLE
Date conversion bug in Journal app

### ISSUE THREAD

**Issue by b1conrad** [Picolab/Manifold#185]
There is a function `convertDate` that is defined in two different places [here](https://github.com/Picolab/Manifold/tree/master/src/components/Apps/Picolabs/Journal). Two problems with this code: 

1. If the time change required (for time zone of owner) changes the date, this code doesn't compute the correct date.
2. When this is fixed, it will have to be done in both places (so maybe there should only be one place?)

Another possibility would be to use code that already exists, rather than rolling our own (and risking this kind of problem), for example,  either `moment` or `strftime` ought to provide what we need.

**Comment by windley** [comment]
Yes, the built-in functions are preferable. Was there a reason they didn't work?

**Comment by KandareJ** [comment]
There is no good reason for writing our own function. It was my first manifold app and the convertDate function was from the first stack overflow response on how to convert a timestamp that I found. I'm not sure how that code got duplicated. I assigned myself and I'll fix it.
